importScripts/sampleRecords/library.source.json

```diff
@@ -1,1353 +1,1547 @@
-  "Maps": "0",
-  "Marks": "4p.",
-  "Pages": "4",
-  "Collation": "4p.",
-  "Rare": "0",
-  "Restrictions": "NOT FOR LOAN",
-  "DateDue": "2007-09-27",
-  "DateOut": "2007-08-30",
-  "Borrower": "NOT FOR LOAN",
-  "Value": "0",
-  "Purchased": "no",
-  "Donated": "no"
- },
- {
-  "PublicationID": "844",
-  "Title": "Shark Bay through four centuries 1616 to 2000",
-  "LegacyID": "844",
-  "Series": "0",
-  "Authors": "Edwards, Hugh",
-  "LastName": "Edwards",
-  "FirstNames": "Hugh",
-  "HomeLocation": "LIBRARY",
-  "ISBN": "415",
-  "LastEditBy": "JILLM",
-  "LastEditDate": "2003-06-30 00:00:00",
-  "LibraryNo": "EDW",
-  "PublicationType": "Book",
-  "PublicationYear": "1999",
-  "Publisher": "Shire of Shark Bay",
-  "Subject": "Shark Bay",
-  "Illustrations": "1",
-  "Maps": "0",
-  "Marks": "415p. Illus., photos",
-  "Pages": "0",
-  "Collation": "415p. Illus., photos",
-  "Rare": "0",
-  "Restrictions": "NOT FOR LOAN",
-  "DateDue": "2002-05-17",
-  "DateOut": "2002-04-19",
-  "Borrower": "NOT FOR LOAN",
-  "Value": "0",
-  "Purchased": "no",
-  "Donated": "no"
- },
- {
-  "PublicationID": "88",
-  "Title": "Official record of the debates. Third session. Melbourne, 20th Jan to 17 March, 1898",
-  "LegacyID": "88",
-  "Series": "0",
-  "Authors": "Australasian Federal Convention",
-  "LastName": "Australasian Federal Convention",
-  "HomeLocation": "LIBRARY",
-  "LastEditBy": "LIBRARY",
-  "LastEditDate": "2012-02-10 00:00:00",
-  "LibraryNo": "RB  AUS",
-  "Notes": "2 vols.",
-  "PublicationType": "Book",
-  "PublicationYear": "1898",
-  "Publisher": "Govt. Printer",
-  "WherePublished": "Melbourne",
-  "Subject": "Federation; Australian Federal Convention",
-  "CurrentLocation": "Rare book shelves",
-  "Illustrations": "0",
-  "Maps": "0",
-  "Pages": "0",
-  "Rare": "1",
-  "Restrictions": "NOT FOR LOAN",
-  "DateDue": "2002-02-21",
-  "DateOut": "2002-01-24",
-  "VolumeNum": "2 vols.",
-  "Borrower": "NOT FOR LOAN",
-  "Value": "0",
-  "Purchased": "no",
-  "Donated": "no"
- },
- {
-  "PublicationID": "9",
-  "Title": "Merredin; a photographic history",
-  "LegacyID": "9",
-  "Series": "0",
-  "Size": "Large",
-  "Authors": "Abell, Bob; Abell, Barbara",
-  "LastName": "Abell",
-  "FirstNames": "Bob",
-  "OtherAuthors": "Abell, Barbara",
-  "HomeLocation": "LIBRARY",
-  "LastEditBy": "LIBRARY",
-  "LastEditDate": "2013-02-19 00:00:00",
-  "LibraryNo": "Q  ABE",
-  "Notes": "Large book section\r\n",
-  "PublicationType": "Book",
-  "Subject": "Merredin - Pictorial works",
-  "CurrentLocation": "Large book shelves",
-  "Illustrations": "1",
-  "Maps": "0",
-  "Marks": "151p. Illus., photos.",
-  "Pages": "151",
-  "Collation": "151p. Illus., photos.",
-  "Rare": "0",
-  "Restrictions": "NOT FOR LOAN",
-  "DateDue": "2002-02-14",
-  "DateOut": "2002-01-17",
-  "Borrower": "NOT FOR LOAN",
-  "Value": "0",
-  "Purchased": "no",
-  "Donated": "no",
-  "Review": "RWAHS newsletter,, Vol. 31, no.10, November 1992, p.8 / M.H."
- }
+[
```

Comment:
similar whitespace difference to previous PR. You can ignore whitespace using ?w=1 in the URL.